### PR TITLE
Cloud service for medusa-metrics-sink

### PR DESCRIPTION
**Pull request by ddhirajkumar**
[STRMALERT-3219]
Cloud service for medusa-metrics-sink

### CODE REVIEW

**Comment by mosabua** [comment]
The problem with the CLA signing is that you might have signed the CLA with the github.com user account but all your commits are done with a different user id. Probably the internal one. Since you have reached out to me internally and I have checked you are okay to go with this PR but going forward please make sure you configure your git client properly so commits use the correct username...

**Comment by ddhirajkumar** [comment]
@mosabua 
Thanks for the info. I have setup my email address using this command on my local
*git config user.email "dhiraj@gmail.com"* 
That seems to have helped. I shall verify it for the next PR

File: components/cookbooks/medusa-metrics-sink/metadata.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  :packages => [ 'mgmt.cloud.service', 'cloud.service'  ],
+  :namespace => true
+
+attribute 'url',
+  :grouping => 'service',
+  :description => "Kafka Broker URL",
+  :required => "required",
+  :default => '',
+  :format => { 
+    :help => 'Kafka Broker URL (for e.g: kafka.kafka-cluster-telemetry.prod1-ndc.walmart.com:9092)',
```
Comment (dandagopi) [inline components/cookbooks/medusa-metrics-sink/metadata.rb]: @ddhirajkumar please remote in example Walmart  related urls

Comment (ddhirajkumar) [reply components/cookbooks/medusa-metrics-sink/metadata.rb]: Sure will change and update the PR

**Pull request closed without merging**